src/stdlib/math.rs

```diff
@@ -18,9 +20,30 @@ pub fn new() -> Module {
             "nan".into() => Type::Float(f32::NAN),
 
             // funcs
-            "sin".into() => native_func!(|_, _args| {
-                Ok(Type::Float(3f32))
-            }, 1)
+            // TODO: add more functions
+            // Trigonometric functions
+            "sin".into() => native_func!(|_, args| {
+                Ok(Type::Float(args[0].parse::<f32>()
+                    .expect("Invalid Argument: Type is not Float").sin()))
+            }, 1),
+            "cos".into() => native_func!(|_, args| {
+                Ok(Type::Float(args[0].parse::<f32>()
+                    .expect("Invalid Argument: Type is not Float").cos()))
+            }, 1),
+            "tan".into() => native_func!(|_, args| {
+                Ok(Type::Float(args[0].parse::<f32>()
+                    .expect("Invalid Argument: Type is not Float").tan()))
+            }, 1),
+
+            // RNG functions
+            "randrange".into() => native_func!(|_, args| {
+                Ok(Type::Float(thread_rng().gen_range(
```

Comment:
Ideally, `thread_rng` would be called once per *thread*, as per its design, instead of instantiating it every time it is required. If this isn't possible (and it probably isn't with the current module system), using the standard `rand::random` function may be more idiomatic.

Reply:
Oh, yeah. I haven't really delve too deep into the rand library, so I only have surface-level knowledge about it. Also will do.

Reply:
After further reading, I don't think that replacing gen_range is possible with rand::random?

Reply:
As I recommended, that would be the best option if you're not able to instantiate a single instance of `thread_rng` at initialisation.